static/js/actions/channel.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+import { createAction } from "redux-actions"
+
+export const SET_CHANNEL_DATA = "SET_CHANNEL_DATA"
+export const setChannelData = createAction(SET_CHANNEL_DATA)
```

Comment:
Any reason these shouldn't just be defined in `static/js/actions/channels.js`? The reducer is defined in `static/js/reducers/channels.js` after all